### ISSUE TITLE
Streaming from bluetooth stops after XX minutes or because of "missing timestamp"

### ISSUE THREAD

**Issue by sorenopsy** [alexandrebarachant/muse-lsl#46]
Muse-lsl, latest version at time of writing
OS: Ubuntu 16.04 adm 64 bit
Bluetooth:Internal Bluetooth of Lenovo T450
pygatt 3.1.

I've been able to stream data to lsl-viewer, but after 2 minutes or when there is a missing sample the stream just stop.
With hcidump i can see that the "missing sample" stop the transfer of data from the muse to the bluetooth device.

**Comment by sorenopsy** [comment]
Addendum:
I've been able to stream to openvibe by changing the chunk size on muse-lsl.my, but after 2 minutes or when there is a missing sample the stream just stop.
With hcidump i can see that the "missing sample" stop the transfer of data from the muse to the bluetooth device.

**Comment by sorenopsy** [comment]
I've discovered the problem is with Internal Blueooth of Lenovo T450.
So i've disabled the internal one and  bought a Trust Bluetooth Dongle (chipset CSR8510 )
which is also suggested by muse
http://developer.choosemuse.com/hardware-firmware/bluetooth-connectivity
Now it works just fine, no missing packets and no timeouts.
I've tried both under linux with lsl viewer and openvibe and under windows with Bluemuse.
